src/screens/DiscoveryScreenV2/fragment/DomainFragment.js

```diff
@@ -251,11 +252,11 @@ const styles = StyleSheet.create({
   },
   fragmentContainer: {
     flex: 1,
-    backgroundColor: COLORS.white
+    backgroundColor: colors.white
   },
   noDataFoundContainer: {
     flex: 1,
-    backgroundColor: COLORS.white,
+    backgroundColor: colors.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color properties have been updated correctly from `COLORS.white` to `colors.white`. However, please ensure that the new `colors` module provides the same color values as the old `COLORS` module to maintain consistency in the UI.
```diff
-    backgroundColor: COLORS.white
+    backgroundColor: colors.white
```
```diff
-    backgroundColor: COLORS.white,
+    backgroundColor: colors.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->